client/src/locales/en.json

```diff
@@ -216,6 +216,10 @@
   "dashboard.index.no-arch-sacks": "There are no archived sacks yet.",
   "dashboard.index.no-sacks": "There are no sacks yet.",
   "dashboard.index.show-arch-sacks": "Show archived sacks",
+  "delete-link.has-expired": "Delete account link has expired. If you still want to delete account go to {link} page.",
+  "delete-link.has-expired-link": "user profile page",
+  "email.delete-account.failure": "There was an error while sending email.",
+  "email.delete-account.success": "An email containing a link that will allow you to delete your account has been sent.",
```

Comment:
I think that 'allow' word might not be appropriate. Maybe just change to: 'We sent you an email with instructions to delete your account.' or 'Check your email for a link to delete your account. If it doesn’t appear within a few minutes, check your spam folder.'